core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -795,22 +795,9 @@ class ZkUtils(val zkClient: ZkClient,
     * client updates the path stat.version gets incremented
     */
   def getSequenceId(path: String, acls: java.util.List[ACL] = DefaultAcls): Int = {
-    try {
+      makeSurePersistentPathExists(path)
```

Comment:
As of now, getSequenceId method is called only once.  I think current PR is simple and easy to understand. I can change if any concern.